src/Templates/src/templates/maui-blazor/MauiApp.1.csproj

```diff
@@ -29,10 +29,9 @@
 
 	<ItemGroup>
 		<!-- App Icon -->
-		<MauiImage
+		<MauiAppIcon
 			Include="Resources\appicon.svg"
 			ForegroundFile="Resources\appiconfg.svg"
-			IsAppIcon="true"
 			Color="#512BD4" />
```

Comment:
The template is updated, so we probably also need to update docs and things. Not sure who is the best to talk to.